enterprise/server/util/distributed_client/distributed_client.go

```diff
@@ -34,6 +34,8 @@ const (
 	// Keep under the limit of ~4MB (save 256KB).
 	// (Match the readBufSizeBytes in byte_stream_server.go)
 	readBufSizeBytes = (1024 * 1024 * 4) - (1024 * 256)
+
+	writeBufSizeBytes = 512 * 1024 // 512 KiB
```

Comment:
Probably worth a comment explaining what this is (and what the reasoning is behind this choice of value - could just be "seems to work well enough for now")